activesupport/lib/active_support/testing/method_call_assertions.rb

```diff
@@ -30,6 +27,26 @@ def assert_called_with(object, method_name, args = [], returns: nil)
         def assert_not_called(object, method_name, message = nil, &block)
           assert_called(object, method_name, message, times: 0, &block)
         end
+
+        def assert_any_instance_called(klass, method_name, times: 1, returns: nil)
+          times_called = 0
+
+          klass.class_eval { define_method(method_name) { |*args| times_called += 1; returns } }
```

Comment:
Won't we need to bring the old method back? We should definitely test that a method going through the ringer here comes back out the way it was :smile: